package.json

```diff
@@ -47,13 +44,29 @@
     "lodash": "^4.17.19",
     "resin-cli-form": "^2.0.2",
     "resin-cli-visuals": "^1.7.0",
-    "resin-discoverable-services": "git+https://github.com/resin-io-modules/resin-discoverable-services.git#find-on-all-interfaces",
+    "resin-discoverable-services": "^2.0.3",
```

Comment:
Did we discuss this change, perhaps in Flowdock, when the PR was being reviewed 2 months ago? I can't say that I remember. :-)  I've noticed this now while reviewing another PR, balena-io/balena-cli#2454. There are logic differences between `resin-discoverable-services` v2.0.3 and branch `resin-discoverable-services.git#find-on-all-interfaces`. Here's the diff: https://github.com/balena-io-modules/resin-discoverable-services/compare/v2.0.3...find-on-all-interfaces

The commit of branch `find-on-all-interfaces` has this description:

```
Specifically listen on any valid IPv4 interface, to ensure we get responses from all NICs back.

* Filters responses so there are no duplicates.
* Uses modified `bonjour`->`multicast-dns` branches (experimental) to perform multi-nic listen.

WARNING: This should not be merged onto `master`, as more work needs to be done to the branches.
```

Despite that warning message in the commit description, the balena CLI has been using that branch in the `balena scan` command for more than 4 years (it predates me!), and to a lesser extent in the `balena join` and `balena leave` commands.